CI throws a random-looking error

## 📝 Provide detailed reproduction steps (if any)

Since a few hours, CI throws a random-looking error: https://app.travis-ci.com/github/ckeditor/ckeditor5/jobs/581814439

<details><summary>See the log</summary>

<pre>
Compiling ckeditor5-engine

* yarn run build --sourceMap
error Command failed with exit code 2.
child_process.js:866
    throw err;
    ^

Error: Command failed: yarn run build --sourceMap
error Command failed with exit code 2.

    at checkExecSyncError (child_process.js:790:11)
    at Object.execSync (child_process.js:863:15)
    at checkPackagesCodeCoverage (/home/travis/build/ckeditor/ckeditor5/scripts/ci/check-packages-code-coverage.js:72:16)
    at Object.<anonymous> (/home/travis/build/ckeditor/ckeditor5/scripts/ci/travis-check.js:27:2)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
    at internal/main/run_main_module.js:17:47 {
  status: 2,
  signal: null,
  output: [
    null,
    Buffer(1517) [Uint8Array] [
      121,  97, 114, 110,  32, 114, 117, 110,  32, 118,  49,  46,
       50,  50,  46,  49,  57,  10,  36,  32, 116, 115,  99,  32,
       45, 112,  32,  46,  47, 116, 115,  99, 111, 110, 102, 105,
      103,  46, 114, 101, 108, 101,  97, 115, 101,  46, 106, 115,
      111, 110,  32,  45,  45, 115, 111, 117, 114,  99, 101,  77,
       97, 112,  10,  46,  46,  47,  46,  46,  47, 110, 111, 100,
      101,  95, 109, 111, 100, 117, 108, 101, 115,  47,  64, 116,
      121, 112, 101, 115,  47,  99,  97,  99, 104, 101,  97,  98,
      108, 101,  45, 114,
      ... 1417 more items
    ],
    Buffer(39) [Uint8Array] [
      101, 114, 114, 111, 114,  32,  67, 111,
      109, 109,  97, 110, 100,  32, 102,  97,
      105, 108, 101, 100,  32, 119, 105, 116,
      104,  32, 101, 120, 105, 116,  32,  99,
      111, 100, 101,  32,  50,  46,  10
    ]
  ],
  pid: 8457,
  stdout: Buffer(1517) [Uint8Array] [
    121,  97, 114, 110,  32, 114, 117, 110,  32, 118,  49,  46,
     50,  50,  46,  49,  57,  10,  36,  32, 116, 115,  99,  32,
     45, 112,  32,  46,  47, 116, 115,  99, 111, 110, 102, 105,
    103,  46, 114, 101, 108, 101,  97, 115, 101,  46, 106, 115,
    111, 110,  32,  45,  45, 115, 111, 117, 114,  99, 101,  77,
     97, 112,  10,  46,  46,  47,  46,  46,  47, 110, 111, 100,
    101,  95, 109, 111, 100, 117, 108, 101, 115,  47,  64, 116,
    121, 112, 101, 115,  47,  99,  97,  99, 104, 101,  97,  98,
    108, 101,  45, 114,
    ... 1417 more items
  ],
  stderr: Buffer(39) [Uint8Array] [
    101, 114, 114, 111, 114,  32,  67, 111,
    109, 109,  97, 110, 100,  32, 102,  97,
    105, 108, 101, 100,  32, 119, 105, 116,
    104,  32, 101, 120, 105, 116,  32,  99,
    111, 100, 101,  32,  50,  46,  10
  ]
}
</pre>

</details>

### ✔️ Expected result

CI works.

### ❌ Actual result

CI throws an error.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

Building the `ckeditor5-engine` locally throws more user-friendly error:

```
$ tsc -p ./tsconfig.release.json
../../node_modules/@types/cacheable-request/index.d.ts:26:42 - error TS2709: Cannot use namespace 'ResponseLike' as a type.

26         cb?: (response: ServerResponse | ResponseLike) => void
                                            ~~~~~~~~~~~~

../../node_modules/@types/cacheable-request/index.d.ts:77:51 - error TS2709: Cannot use namespace 'ResponseLike' as a type.

77             listener: (response: ServerResponse | ResponseLike) => void
                                                     ~~~~~~~~~~~~

../../node_modules/@types/cacheable-request/index.d.ts:81:69 - error TS2709: Cannot use namespace 'ResponseLike' as a type.   

81         on(event: 'response', listener: (response: ServerResponse | ResponseLike) => void): this;
                                                                       ~~~~~~~~~~~~

../../node_modules/@types/cacheable-request/index.d.ts:84:71 - error TS2709: Cannot use namespace 'ResponseLike' as a type.   

84         once(event: 'response', listener: (response: ServerResponse | ResponseLike) => void): this;
                                                                         ~~~~~~~~~~~~

../../node_modules/@types/cacheable-request/index.d.ts:89:51 - error TS2709: Cannot use namespace 'ResponseLike' as a type.   

89             listener: (response: ServerResponse | ResponseLike) => void
                                                     ~~~~~~~~~~~~

../../node_modules/@types/cacheable-request/index.d.ts:95:51 - error TS2709: Cannot use namespace 'ResponseLike' as a type.   

95             listener: (response: ServerResponse | ResponseLike) => void
                                                     ~~~~~~~~~~~~

Found 11 errors in the same file, starting at: ../../node_modules/@types/cacheable-request/index.d.ts:26
```

--- a/package.json
+++ b/package.json
@@ -96,6 +96,7 @@
     "@ckeditor/ckeditor5-track-changes": ">=28.0.0",
     "@webspellchecker/wproofreader-ckeditor5": "^2.0.1",
     "@wiris/mathtype-ckeditor5": "^7.24.0",
+    "assert": "^2.0.0",
     "babel-standalone": "^6.26.0",
     "chalk": "^4.1.0",
     "cli-table": "^0.3.1",

--- a/package.json
+++ b/package.json
@@ -113,7 +113,6 @@
     "minimist": "^1.2.5",
     "mkdirp": "^1.0.4",
     "nyc": "^15.0.1",
-    "popper": "^1.0.1",
     "postcss-loader": "^4.3.0",
     "progress-bar-webpack-plugin": "^2.1.0",
     "raw-loader": "^4.0.1",

